app/api/v2/__init__.py

```diff
@@ -0,0 +1,14 @@
+from aiohttp import web
+
+
+def make_app(services):
+    from .security import authentication_required_middleware_factory
```

Comment:
I wrote this and the handler import below inside the `make_app` function to avoid any possible circular imports (which don't actually exist at the moment). I could make a separate module for all this and move it out of the __init__ module, which might reduce the circular import concerns. We could also do that later if wanted.